homeassistant/components/xiaomi_miio/const.py

```diff
@@ -197,9 +197,11 @@ class SetupException(Exception):
 )
 
 # TODO: use const from pythonmiio once new release with the constant has been published. # pylint: disable=fixme
```

Comment:
Is there something holding up the pythonmiio release with these constants?

Reply:
1. its not merged yet: https://github.com/rytilahti/python-miio/pull/1169
2. There will be no release soon: https://github.com/home-assistant/core/issues/57474#issuecomment-946352575

Reply:
Thanks for the backstory